doc/news/_preparation_next_release.md

```diff
@@ -155,7 +155,21 @@ The text below summarizes updates to the [C (and C++)-based libraries](https://w
 Bindings allow you to utilize Elektra using [various programming languages](https://www.libelektra.org/bindings/readme). This section keeps
 you up to date with the multi-language support provided by Elektra.
 
-### <<Binding1>>
+### JNA
+
+Since internal iterator support for `KeySet` is due to being dropped, the following methods have been removed:
+
+- `Elektra::ksNext`
+- `Elektra::ksCurrent`
+- `Elektra::ksGetCursor`
+- `Elektra::ksSetCursor`
+- `KeySet::next`
+- `KeySet::current`
+- `KeySet::rewind`
+- `KeySet::getCursor`
+- `KeySet::setCursor`
+
+Until internal `KeySet` iterator support has been dropped form native library, `Elektra::ksRewind` is being retained while also being annotated as 'depricated for removal'. The reason is, that we still need to rewind a `KeySet` before passing it to a native plugin via `NativePlugin::set`, `NativePlugin::get` or `NativePlugin::error`.
```

Comment:
```suggestion
Until internal `KeySet` iterator support has been dropped form native library, `Elektra::ksRewind` is being retained while also being annotated as 'depricated for removal'. The reason is, that we still need to rewind a `KeySet` before passing it to a native plugin via `NativePlugin::set`, `NativePlugin::get` or `NativePlugin::error`. _(@tucek)_
```

The build server doesn't complain here, since you change more than a few lines of the release notes. You should still always add at least 1 line that ends with your name (or Github Account), so we know who implemented the new feature (without looking at the commit history).